playbooks/group_vars/all.yml

```diff
@@ -0,0 +1,49 @@
+---
+upstream_repos_clone_folder: "/opt"
+suse_osh_registry_location: "{{ lookup('env','OPENSTACK_HELM_IMAGE_REGISTRY_LOCATION') | default('docker.io', true) }}"
```

Comment:
Note: I don't see why we set up this var, as the registry location is set for developer mode already. It works.

Reply:
I have to check the code for details but iirc those locations are pre-defined to use deployer node registry. This options seems to allow other public or internal registry if needed.

Reply:
The extra vars have precedence overall. If you say "I want to use my own registry", then you override that variable in extravars and it works.

Reply:
I don't see that variable currently defined in extravars.

And having that as environment variable sometimes can be easier as user don't have to touch the file.

I don't see obvious side-effect of including this way.